### ISSUE TITLE
This project is a copy of my 450DSA

### ISSUE THREAD

**Issue by AsishRaju** [sounishnath003/450-DSA-Tracker#10]
hey @sounishnath003, I came across  your 450-DSA-Tracker through LinkedIn, 
What are the odds that we build the same kind of project with exactly the same UI/UX and also the same text on the about page?

**My Github Repo: https://github.com/AsishRaju/450-DSA (which is 2 months old)
My Deployed Version: https://450dsa.com/**

I got nothing against this, in fact, your free do so but my only concern is you did not credit the person from where you got this idea at least for the UI and I see you using the same dependency `localbase` as I used so can't see anything new here.

Let's hope this was unknowingly done, for which chances are very low.
✌🏻

**Comment by sounishnath003** [comment]
Hi!,
I was thinking about the tracker as an option so that can help me. I started building the app and when to make the scrapper I came to see your tracker, Yes I took some help. But never mean to a bad thing. I knew the `localbase` but really use `question json.` for yours.  I also started adding a few features to the app and making a feature pack tracker and it will help everyone to practice and help.

**Comment by AsishRaju** [comment]
> hey @sounishnath003, I came across your 450-DSA-Tracker through LinkedIn,
> What are the odds that we build the same kind of project with exactly the same UI/UX and also the same text on the about page?
> 
> **My Github Repo: https://github.com/AsishRaju/450-DSA (which is 2 months old) My Deployed Version: https://450dsa.com/**
> 
> I got nothing against this, in fact, your free do so but my only concern is you did not credit the person from where you got this idea at least for the UI and I see you using the same dependency `localbase` as I used so can't see anything new here.
> 
> Let's hope this was unknowingly done, for which chances are very low.
> ✌🏻

**Here are some proof**

**1.**
**Yours:**
![image](https://user-images.githubusercontent.com/41658324/103296111-6a203480-4a1b-11eb-816e-539f041e66d3.png)
**Mine:**
![image](https://user-images.githubusercontent.com/41658324/103296157-845a1280-4a1b-11eb-9fd6-e03952129ff3.png)

_same text in the topic card, same hover animation, the same progress bar at last same emoji 😑_


**2.**
**Yours**
![image](https://user-images.githubusercontent.com/41658324/103296390-034f4b00-4a1c-11eb-8fdf-1735b4c2f6fd.png)

**Mine**
![image](https://user-images.githubusercontent.com/41658324/103296411-0d714980-4a1c-11eb-9d45-54541e54813e.png)

_same text_


**3.**
**Yours**
![image](https://user-images.githubusercontent.com/41658324/103296468-27ab2780-4a1c-11eb-8c82-736fe22083cf.png)
**Mine**
![image](https://user-images.githubusercontent.com/41658324/103296482-309bf900-4a1c-11eb-9143-c1e40c80a9f9.png)

_same navigation style_

**Comment by sounishnath003** [comment]
Yes I told you already!.

**Comment by AsishRaju** [comment]
> Hi!,
> I was thinking about the tracker as an option so that can help me. I started building the app and when to make the scrapper I came to see your tracker, Yes I took some help. But never mean to a bad thing. I knew the `localbase` but really use `question json.` for yours. I also started adding a few features to the app and making a feature pack tracker and it will help everyone to practice and help.
> 
> But don't take me wrong! I'm not hurting you and feel bad.

yeah bro I understand you took help from my project that completely fine, but you could have at least credited me and tell you got help from here. even i am not trying to be bad here and completely clean your work, but credits matter just imagine someone taking your project and making it exactly /almost the same and giving you credit

**Comment by sounishnath003** [comment]
I appologise. I'll update your profile as also attribution!. And please brother.. if you want to contribute, I welcome and made the platform famous for all, this will help everyone.
also I started finding your linkedin to ask, but could not find you anywhere

**Comment by AsishRaju** [comment]
> If I made it wrong way , I appologise. I'll update your profile as also attribution!. And please brother.. if you want to contribute, I welcome and made the platform famous for all, this will help everyone.
> also I started finding your linkedin to ask, but could not find you anywhere

No bro i don't have any intension to pull this long, a proper credit will always do the work, my LinkedIn is easily available through my GitHub profile, and also i have linked my LinkedIn in my 450dsa.com about page also so..

**Comment by sounishnath003** [comment]
Please accept my connection request

**Comment by AsishRaju** [comment]
> Please accept my connection request

done

**Comment by sounishnath003** [comment]
ok. I am adding your attribution

**Comment by kakashi777** [comment]
The best part about all this is he denies copying anything with a straight face and adds a copyright in the footer of the app😂

**Comment by kakashi777** [comment]
Also, might as well have copied it in JS, you clearly cannot write typescript (but good attempt to hide copyting), all you can write is basic interface.

**Comment by kakashi777** [comment]
Good attempt at changing the issue title to remove the word "copy"...disgrace

**Comment by Shashank-Deep** [comment]
Any one plz help me how to run this reo in local system
